python/mcap/mcap0/chunk_builder.py

```diff
@@ -27,7 +27,7 @@ def add_schema(self, schema: Schema):
         schema.write(self.record_writer)
 
     def add_message(self, message: Message):
-        if self.message_start_time == 0:
+        if self.num_messages == 0:
```

Comment:
This is not correct, at least on its own. The Chunk start_time is the minimum timestamp in the Chunk, not the first timestamp (unless those happen to be the same).

Reply:
That’s correct. It’s the range of the log_times contained in the chunk, regardless of their in-chunk ordering.

Reply:
Updated logic to use min & max message time to set chunk start & end times.